### ISSUE TITLE
I can't turn off notifications

### ISSUE THREAD

**Issue by Leo-Vargas** [Seldszar/Gumbo#113]
**Describe the bug**
I had notifications turned on only to a couple streams, but all of a sudden it turned on for all the streams i follow and i can't turn it off. When i click the button to do so nothing happens. 

**To Reproduce**
1. Go to 'settings' then 'notifications'
2. Click on any of the buttons and nothing will change.

**Expected behavior**
I expected the button to go from enabled to disabled. If i change tabs (from 'notifications' to 'badge' and back to 'notifications', for exemple) the button seems to change state, but when i refresh the page it goes back to its original state and i keep getting notifications.

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Brave
 - Version: 1.46.134

**Comment by Tichael** [comment]
I want to add on that since I have the same bug. I had notification disabled ever since I installed the extension, and they just enabled themself. When I try to disable them, I have the same bug where clicking on the "switch" doesn't do anything. I noticed that, if you click on it, change page in the settings then go back, it did switched correctly, but closing and reopening the settings shows that it didn't save.

I can also see that no settings can be changed, so it's not just the notification settings.
